Update BF CF docs (#646)
RedisBloom/RedisBloom: docs/commands/cf.scandump.md

Begins an incremental save of the cuckoo filter.

> This command is useful for large cuckoo filters which cannot fit into the normal `DUMP` and `RESTORE` model.

Reviewer: ```suggestion
This command is useful for large cuckoo filters that cannot fit into the `DUMP` and `RESTORE` model.
```
